samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -11,8 +11,8 @@
 #define BIG_TERMINATE_TIMEOUT 60 /* seconds */
 
 #define BIS_ISO_CHAN_COUNT 1
-#define ISO_MTU (BT_ISO_CHAN_SEND_RESERVE + sizeof(uint32_t))
-NET_BUF_POOL_FIXED_DEFINE(bis_tx_pool, BIS_ISO_CHAN_COUNT, ISO_MTU, NULL);
+NET_BUF_POOL_FIXED_DEFINE(bis_tx_pool, BIS_ISO_CHAN_COUNT,
+			  BT_ISO_SDU_BUF_SIZE(sizeof(uint32_t)), NULL);
```

Comment:
May be outside the scope of this PR, but the sample should use SDU sizes similar to typical audio usecases. Hopefully, this sample is tested for 0, 4 (as in uint32_t used here), 23, 251 and 4095 (later values from Kconfig range).

Reply:
ISO != Audio, so I disagree. 

I did consider using `CONFIG_BT_ISO_TX_MTU` instead of `sizeof(uint32_t)` for the buffer size as well.

Reply:
If the sample is using 4 bytes, then the MTU should be set to 23 bytes, using 251 bytes as default will show higher RAM usage.